test/mariner2_test.go

```diff
@@ -24,7 +23,7 @@ func testLinuxDistro(ctx context.Context, t *testing.T, buildTarget string) {
 		t.Parallel()
 		spec := dalec.Spec{
 			Name:        "test-build-commands-fail",
-			Version:     "v0.0.1",
+			Version:     "0.0.1",
```

Comment:
why is the drop of the `v` prefix?

Reply:
It's not technically correct.
Also, IIRC, even though rpmbuild doesn't seem to care, debbuild didn't like it.